### PR TITLE
Improve block cursor's visibility.

### PR DESCRIPTION
**Pull request by aioutecism**
Address #2943.

### CODE REVIEW

File: src/vs/editor/browser/widget/media/editor.css [hunk 1]
```diff
@@ -119,6 +119,7 @@
 .monaco-editor .view-lines {
 	cursor: text;
 	white-space: nowrap;
+	z-index: 1;
```
Comment (aioutecism) [inline src/vs/editor/browser/widget/media/editor.css]: This makes text rendered over cursor.

**Comment by alexandrudima** [comment]
@aioutecism Thank you for the contribution. The problem is that even if the character is rendered on top, it is still possible that the cursor is rendered in black and the character is rendered in black, so opacity or z-index cannot solve this issue alone.

I have just made a change in 9a88999b599723d1ad1df42701f0ce4731a231e4 where the block cursor renders inside itself the character it sits on in the opposite cursor color:

block

**Comment by aioutecism** [comment]
Thanks! Your implementation is cool!

**Comment by mbroadst** [comment]
@alexandrudima did this make it into the insiders build 0.10.15? I don't see a difference there

**Comment by aioutecism** [comment]
No. You can try it on master branch.

**Pull request closed without merging**